hugo_site/content/live.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Schedule
+date: 2018-12-16T18:48:39+01:00
+draft: false
+type: "single"
+description: "DjangoCon Europe 2019 LIVE - stream and ask questions"
+menu:
+  main:
+    parent: "schedule"
+    name: "Live stream and questions"
+---
+
+# Live stream
+
+Tune in here: https://www.youtube.com/user/djangoconeurope
+
+# Questions
+
+During talks, a host will select questions from the internet crowd. You can tag your question
```

Comment:
"Attendees can also use the designated Slack channel.", no?

Reply:
Well, I was thinking that the questions are NOT for attendees :) They still do ask at the microphone?

Reply:
My concern would be about creating a virtual interaction that sucks out the life of the actual IRL interactions.

Reply:
I'd say if somebody is already checking IRC, they could also check a Slack channel. I'm sure there are attendees who are more comfortable asking a question in a channel than asking it at a microphone.

Reply:
I am in favour of only doing questions through slack/twitter/IRC, because it allows pre-moderating all questions. We did this for DUTH too.

Allowing random people to ask questions can easily lead to questions that aren't questions, questions that are way too long, too detailed, meant to trick the speaker into looking bad or questions only interesting to 2 people in the audience. DjangoCon has not been as bad with this traditionally though.

Having a moderator in between all questions allows the moderator to pick out questions that are actually interesting for the wider audience, and questions that help speakers shine rather than drive them into a corner about some irrelevant contradiction in their talk. It is a bit more work for the moderator though.

And as Tobias said, standing up at a microphone can be intimidating, and established people tend to be more comfortable asking than first time attendees.

A potential downside is that it's hard to ask for clarification on a question, but if that's needed, perhaps the question is better handled off-line anyways.

Reply:
@mxsasha 
As a new conference organizer, I feel uncomfortable making choices that alter the conference too much from the usual format. In this case if we said "this year we don't have questions from microphones", that would be a big change. I would personally find it hard to justify because I *LOVE* questions from the crowd :) Especially @freakboy3742 's :) I think that it's important to maintain the question culture like many Django-nauts do by being polite and gentle.

But you are ALSO about everything else! So maybe the opportunity here is this:

* Writing that we would LOVE for people streaming talks to ask questions: Be precise, ask in due time, withdraw your question loudly if you find that it gets answered in the talk.
* Writing that attendees are ALSO welcome to ask questions in the open online channels, but that they are encouraged to ask them at the microphone
* Always ask good questions!
* A good question: Is one that seeks to explore the topic of the talk; is perceived valuable to others in the audience; clarifies something that you found hard to understand in the talk.
* A bad question: Is something that seeks to emphasize your own knowledge; seeks details at a level that is irrelevant to most people; or tries to trick the speaker, make them look bad or point out errors.

@rixx 
Ah yes, actually your point is really good: These words were actually intended for people streaming the talks and not the attendees at all. IRC is (in theory) open to anyone without signing up, while the attendee Slack is deliberately closed. I'll try to change the text so it doesn't confuse attendees.

Reply:
That sounds good, @benjaoming, it makes asking questions more accessible, while keeping some of the good parts. Like I said, DjangoCon's microphone questions are usually quite good. I don't know if your descriptions of "good question" and "bad question" are quick ideas jotted down, or meant to be final wordings? Because I think they can be tweaked a bit.

Reply:
@mxsasha they were indeed something quickly written, but I will commit it for further reviewing and we can find the right wording if you have time -- am thinking to get this merged within the next 2 days, and we will probably copy paste the text into a printed program, too :)